app/src/main/java/com/google/droidjump/LevelsFragment.java

```diff
@@ -35,17 +37,16 @@
 public class LevelsFragment extends Fragment {
 
     private MainActivity activity;
-    private int[] levels;
     private LevelsAdapter adapter;
 
     @Override
     public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         activity = Objects.requireNonNull((MainActivity) getActivity());
-        levels = loadLevels();
+        int[] levels = loadLevels();
```

Comment:
You don't save levels to a class field "levels" this way. Is this intended? If yes, do you still need this field?

Reply:
You want me to move loadLevels() to adapter and delete int[] levels from constructor?